pythonosc/osc_server.py

```diff
@@ -44,14 +44,14 @@ def _is_valid_request(request: List[bytes]) -> bool:
 class OSCUDPServer(socketserver.UDPServer):
     """Superclass for different flavors of OSC UDP servers"""
 
-    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher) -> None:
+    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher, is_listen=True) -> None:
```

Comment:
can you name this `bind_and_activate` as per official docs?
https://docs.python.org/3.6/library/socketserver.html
this way people will know what it is.

Reply:
Nice idea and done